app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/MapScreen.kt

```diff
@@ -123,7 +122,12 @@ fun moveCamera(
  * @param beacons The list of beacons to be displayed on the map.
  */
 @Composable
-fun DisplayBeacons(beacons: List<Beacon>) {
+fun DisplayBeacons(navigationActions: NavigationActions, beacons: List<Beacon>) {
```

Comment:
You can just pass a callback it's cleaner.

Reply:
sorry, but i don't really understand what you mean here?

Reply:
Instead of passing the navigation action, you can passe the navigate to. But it's fine just small thing